server/channels/api4/oauth_test.go

```diff
@@ -78,6 +78,8 @@ func TestCreateOAuthApp(t *testing.T) {
 }
 
 func TestUpdateOAuthApp(t *testing.T) {
+	t.Skip("https://mattermost.atlassian.net/browse/MM-62895")
```

Comment:
I'm wondering if we could have a dedicated function `SkipFlakyTest(t *testing.T, ticketURL string)` so that they are easily "greppable" in the future?

Reply:
Interesting idea. 1/5 is the prefix of `t.Skip("MM-XX` or `t.Skip("https://mattermost.atlassian.net` enough to grep them?

Reply:
Yeah it could be anything, we just have to make sure we are consistent.